R/estimate_dispersion.R

```diff
@@ -47,7 +54,30 @@
 estimate_dispersion <- function(
     pt_nowcast_mat_list,
     trunc_rep_tri_list,
-    n = length(pt_nowcast_mat_list)) {
+    n = length(pt_nowcast_mat_list),
+    fun_to_aggregate = sum,
```

Comment:
If we are already doing the rowsum do we really want this? Don't we instead want an identity here?

Reply:
Yeah this isn't functionally doing anything because it is summing across `k = 1` rows, so it is just the sum of one element. 

I think I mostly just didn't know how to program in a function to do nothing

Reply:
functon that returns the input for the future.